api/src/main/resources/workbench-api.yaml

```diff
@@ -8052,6 +8052,8 @@ components:
       properties:
         html:
           type: string
+        text:
```

Comment:
Added text version of preview in addition to html. The functions are doing nearly the same thing, so it seemed inefficient to make two roundtrips to the api.

Reply:
makes sense, and adding a field should be deploy-safe